pages/instructorAssessmentInstance/instructorAssessmentInstance.ejs

```diff
@@ -78,6 +78,26 @@
                 <% } %>
               </td>
             </tr>
+            <tr>
+              <th>Statistics</th>
+              <td colspan="2">
+                <% if (assessment_instance.include_in_statistics) { %>
+                    Included
+                    <a tabindex="0" class="btn btn-xs" role="button"
+                        data-toggle="popover" data-trigger="focus" data-container="body"
```

Comment:
I'm wondering if a tooltip is better for this kind of scenario, both in terms of simplicity and accessibility.

Reply:
What's the accessibility benefit here? (not disagreeing, just curious)

Reply:
I don't have a lot of expertise here, but it would make sense that an element that has a tooltip role would more clearly be parsed by a screen reader than a popover without a clear indication of its intended purpose. Though I was trying to read about it, and I may be wrong: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/tooltip_role

Reply:
Thanks for clarifying. I'll leave this as is for now because I'm also not super sure what the best option is here.